neo/rawio/blackrockrawio.py

```diff
@@ -486,23 +493,16 @@ def _parse_header(self):
         # block_ann['avail_sif'] = self._avail_files['sif']
         # block_ann['avail_ccf'] = self._avail_files['ccf']
         block_ann['rec_pauses'] = False
-
-        for c in range(unit_channels.size):
-            unit_ann = self.raw_annotations['unit_channels'][c]
-            channel_id, unit_id = self.internal_unit_ids[c]
-            unit_ann['channel_id'] = self.internal_unit_ids[c][0]
-            unit_ann['unit_id'] = self.internal_unit_ids[c][1]
-            unit_ann['unit_tag'] = {0: 'unclassified', 255: 'noise'}.get(unit_id, str(unit_id))
-            unit_ann['description'] = 'Unit channel_id: {}, unit_id: {}, unit_tag: {}'.format(
-                channel_id, unit_id, unit_ann['unit_tag'])
-
+        
+        # this is not used anymore because not more ChannelIndex
```

Comment:
Then it is better to remove it.

Reply:
@samuelgarcia Why do you want to keep this documented as a string in the code? Are you planning to reintroduce this at a later time point?

Reply:
I think it could be usefull to keep for a while Maybe we can propagate this annoations to AnalogSignal if we need.
One day this blackrock need a big big clean.
That day we can remove, no ?

Reply:
Ok, then we keep it for now...